### DISCUSSION TITLE
How do I set the owner of popup

### DISCUSSION THREAD

**Discussion by chenfeng11** [AvaloniaUI/Avalonia#12983] [Q&A]
I want to use popup ,But it will be placed on top of other applications.I don't know how to set its owner.I need some help

**Comment by maxkatz6** [comment]
> But it will be placed on top of other applications

You can't. You don't have control over other applications.

**Reply by chenfeng11** [reply]
You mean I can't set the owner of popup？

**Reply by maxkatz6** [reply]
You can set owner of Popup.
```csharp
var popup = new Popup();
popup.Target = myWindow; // or any control
popup.IsOpen = true;
```

But popup Target can only be part of the same application.

**Reply by timunie** [reply]
We have OverlayPopups, which may not be rendered outside Avalonias Window-bounds, or am I wrong?

**Reply by timunie** [reply]
```diff
    public static AppBuilder BuildAvaloniaApp()
        => AppBuilder.Configure<App>()
            .UsePlatformDetect()
            .WithInterFont()
            .LogToTrace()
+           .With(new Win32PlatformOptions(){ OverlayPopups = true})
            .UseReactiveUI();
```
